Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/PublicBlogController.java

```diff
@@ -0,0 +1,102 @@
+package edu.asu.spring.quadriga.web.publicwebsite;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+
+@PropertySource(value = "classpath:/user.properties")
+@Controller
+/**
+ * This controller class is responsible for providing views for project blog creation.
+ *
+ * @author Pawan Mahalle
+ *
+ */
+public class PublicBlogController {
```

Comment:
Please read this and adjust formatting
https://diging.atlassian.net/wiki/pages/viewpage.action?pageId=8781898